src/modules/username.rs

```diff
@@ -67,7 +73,17 @@ fn is_login_user(context: &Context, username: &str) -> bool {
         .map_or(true, |logname| logname == username)
 }
 
-#[cfg(target_os = "windows")]
+#[cfg(all(target_os = "windows", not(test)))]
+fn is_root_user() -> bool {
+    use deelevate::{PrivilegeLevel, Token};
+    let token = Token::with_current_process().unwrap();
+    matches!(
+        token.privilege_level().unwrap(),
```

Comment:
I also just noticed that you're using `unwrap` here. Can you change this to soft-fail with `log::warn!`?

Reply:
According to the function definition, `privilege_level` will only return one of the 3 PrivilegeLevel enum values. How about we handle the error in the statement above this one (i.e. `let token = Token::with_current_process().unwrap();`)?

Reply:
It looks like both functions can fail, or they wouldn't be returning a `Result`. Please use something like this for both `unwrap`s:
```rs
let token = match Token::with_current_process() {
    Ok(token) => token,
    Err(e) => {
        log::warn!("Failed to get process token: {e:?}");
        return false;
};
```

Reply:
Done